### PR TITLE
Add note about https redirection

### PR DESCRIPTION
**Pull request by owaincuvelier**
I think my wording is a bit messy but really wanted to jot something down and open a PR for it before I forget. If you have a better one liner for HTTPs redirect please share!

### CODE REVIEW

**Review by shadyvb — APPROVED** [review]

File: docs/https.md [hunk 1]
```diff
@@ -2,12 +2,16 @@
...
 
 HTTPS on Cloud requires user-agents support SNI (Server Name Indication). The following protocols are supported:
 
 - TLS 1.2
 - TLS 1.1
 - TLS 1.0
+
+# HTTPS Redirects
+
+It's common to force all traffic to route via HTTPS. To turn on forced HTTPS Redirection, [submit a support request](docs.altis-dxp.com/guides/getting-help-with-altis/) requesting that HTTPS Redirection be enabled. This is set at the CDN layer.
```
Comment (roborourke) [inline docs/https.md:17]: Not sure if the link works without https://

**Comment by roborourke** [comment]
@owaincuvelier it'll be beneficial to get into the habit of switching to the nightly docs version so your PRs are against the master branch rather than v9 etc...

File: docs/https.md (hunk 1 above)
Comment (roborourke) [reply docs/https.md:17]:
```suggestion
It's common to force all traffic to route via HTTPS. To turn on forced HTTPS Redirection, [submit a support request](https://docs.altis-dxp.com/guides/getting-help-with-altis/) requesting that HTTPS Redirection be enabled. This is set at the CDN layer.
```

**Comment by owaincuvelier** [comment]
@roborourke yeah fair enough. I think I did forget. Nightly is just master, right?

**Comment by roborourke** [comment]
Yep, it's not the end of the world, but ideally better to backport from master back to older versions but we can backport from here to master in this case.

**Comment by owaincuvelier** [comment]
@roborourke that's fine! My bad!

**Pull request merged**